### ISSUE TITLE
Document where hooks are run

### ISSUE THREAD

**Issue by zimme** [matb33/meteor-collection-hooks#65]
Maybe we should document where hooks are run and under which conditions as it isn't clear for people who hasn't grasped the internals of meteor yet.

i.e. When running find/findOne on client results in only find/findOne hooks that are defined on client will run and not the ones on server as it does when calling insert/update/remove and so on.

**Comment by doubletaketech** [comment]
Agree.  It is confusing for people who know Meteor as well. :)  I just asked this same question on the new Meteor forums. :+1:
